scripts/core/eventQuerySynchronizedTimestamps.yml

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (C) 2021-2022 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+#
+# See YaML.md for syntax definition
+#
+--- #--------------------------------------------------------------------------
+type: header
+desc: "Intel $OneApi Level-Zero Extension for supporting event query synchronized timestamps."
```

Comment:
minor. reword to "extension for supporting querying of synchronized event timestamps" ?

Reply:
Thanks for catching that. I meant to rephrase it and remove the word 'synchronized' altogether.